src/stacks-hierarchy/Root_TabNavigatorStack/TabNav_ProfileStack/Profile_DebugInfoScreen.tsx

```diff
@@ -182,6 +179,30 @@ export const Profile_DebugInfoScreen = () => {
       />
       <ScrollView testID="debugInfoScrollView">
         <Section withPadding withTopPadding>
+          <LinkSectionItem
+            text="Restart all onboarding sections"
+            onPress={() => restartAllOnboardingSections()}
+          />
+          <ExpandableSectionItem
+            text="Individual onboarding restarts"
+            showIconText={true}
+            expandContent={onboardingSections.map((onboardingSection) => {
+              const {onboardingSectionId} = onboardingSection;
+              return (
+                <LinkSectionItem
```

Comment:
Using `LinkSectionItem` here doesn't really make sense as we can't have section item inside section item. I know it is just in the debug menu, but still think it should be changed so someone doesn't use this as an example of how to use `ExpandableSectionItem` at a later time.

Reply:
True! Updated